pom.xml

```diff
@@ -1531,7 +1531,7 @@
         <org.apache.httpcomponents.wso2.version>4.3.6.wso2v2</org.apache.httpcomponents.wso2.version>
 
         <!-- Carbon mediation version -->
-        <carbon.mediation.version>4.6.117</carbon.mediation.version>
+        <carbon.mediation.version>4.6.118-SNAPSHOT</carbon.mediation.version>
```

Comment:
Please remove this SNAPSHOT version, once we merge the mediation PR, we can upgrade to the released version